package.json

```diff
@@ -17,21 +21,26 @@
   "watch": {
     "build": "src/*.js"
   },
-  "author": "",
-  "license": "MIT",
+  "dependencies": {
+    "dompurify": "^1.0.10",
+    "insert-text-at-cursor": "^0.1.2",
+    "turndown": "^5.0.3"
```

Comment:
Do these count as dependencies if we roll them up? I'm forever confused by the line in NPM

Reply:
In theory they are dependencies but, yes, since we're including them with roll-up in the final script we can move them to devDependencies.